packages/authentication/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/authentication/classes/GetIdTokenResultResult.java

```diff
@@ -0,0 +1,41 @@
+package io.capawesome.capacitorjs.plugins.firebase.authentication.classes;
+
+import androidx.annotation.Nullable;
+import com.getcapacitor.JSObject;
+import com.google.firebase.auth.GetTokenResult;
+import io.capawesome.capacitorjs.plugins.firebase.authentication.interfaces.Result;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+public class GetIdTokenResultResult implements Result {
+
+    @Nullable
+    private GetTokenResult tokenResult;
+
+    private SimpleDateFormat iso8601Format;
+
+    public GetIdTokenResultResult(@Nullable GetTokenResult tokenResult) {
+        this.tokenResult = tokenResult;
+        this.iso8601Format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+        this.iso8601Format.setTimeZone(TimeZone.getTimeZone("UTC"));
```

Comment:
Remove this and just return the number of milliseconds elapsed since the epoch. Update the types accordingly. Same for iOS.